modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -109,48 +114,13 @@ The Redpanda Cloud *Overview* page lists your bootstrap server URL and security
 
 Serverless clusters work with all Kafka clients. For more information, see xref:develop:kafka-clients.adoc[].
 
-Serverless clusters support all major Apache Kafka messages for managing topics, producing/consuming data (including transactions), managing groups, managing offsets, and managing ACLs. (User management is available in the Redpanda Cloud UI or with `rpk acl`.) The following Kafka messages are supported:
-
-* `Produce`                     
-* `Fetch`                     
-* `ListOffsets`                 
-* `Metadata`                    
-* `OffsetCommit`                
-* `OffsetFetch`                 
-* `FindCoordinator`             
-* `JoinGroup`                   
-* `Heartbeat`                   
-* `LeaveGroup`                  
-* `SyncGroup`                   
-* `DescribeGroups`              
-* `ListGroups`                  
-* `SASLHandshake`               
-* `ApiVersions`                 
-* `CreateTopics`                
-* `DeleteTopics`                
-* `DeleteRecords`               
-* `InitProducerID`              
-* `OffsetForLeaderEpoch`        
-* `AddPartitionsToTxn`          
-* `AddOffsetsToTxn`             
-* `EndTxn`                      
-* `TxnOffsetCommit`             
-* `DescribeACLs`                
-* `CreateACLs`                  
-* `DeleteACLs`                  
-* `DescribeConfigs`             
-* `AlterConfigs`                
-* `AlterReplicaLogDirs`         
-* `DescribeLogDirs`             
-* `SASLAuthenticate`            
-* `CreatePartitions`            
-* `DeleteGroups`                
-* `IncrementalAlterConfigs`
-
-=== Unsupported features
-
-The following features are supported in Redpanda Dedicated and BYOC clusters, but they are not yet supported in Serverless clusters: 
-
-* Redpanda Admin, HTTP Proxy, and Schema Registry APIs (You can use an external schema registry.)
+Serverless clusters support all major Apache Kafka messages for managing topics, producing/consuming data (including transactions), managing groups, managing offsets, and managing ACLs. (User management is available in the Redpanda Cloud UI or with `rpk acl`.) 
+
+==== Unsupported features
+
+Redpanda Serverless currently only supports the Kafka API. The Redpanda Admin, HTTP Proxy, and Schema Registry APIs are not exposed. You can use an external schema registry. 
+
+The following features are not yet supported in Serverless clusters: 
```

Comment:
why are we saying "not yet"?  Sounds like a promise/suggestion about what we might support sometime in the future, which we don't do in the docs. Maybe stick with what is supported for beta and update when that changes?